core/parachain/availability/chunks.hpp

```diff
@@ -6,74 +6,75 @@
 #ifndef KAGOME_PARACHAIN_AVAILABILITY_CHUNKS_HPP
 #define KAGOME_PARACHAIN_AVAILABILITY_CHUNKS_HPP
 
-#include <erasure_coding/erasure_coding.h>
+#include <ec-cpp/ec-cpp.hpp>
 
 #include "parachain/availability/erasure_coding_error.hpp"
 #include "runtime/runtime_api/parachain_host_types.hpp"
 
 namespace kagome::parachain {
   inline outcome::result<size_t> minChunks(size_t validators) {
-    unsigned long out = 0;
-    auto r = ECCR_get_recovery_threshold(validators, &out);
-    if (r.tag != NPRSResult_Tag::NPRS_RESULT_OK) {
-      return ErasureCodingError(r.tag);
+    auto res = ec_cpp::getRecoveryThreshold(10);
+    if (ec_cpp::resultHasError(res)) {
+      return ErasureCodingError(ec_cpp::resultGetError(std::move(res)));
```

Comment:
note: ec-cpp error enum starts with "0 != ok" (this project seems to use `enum Error { first = 1 }`, won't break `outcome`, only thing that breaks is improper `error_code` check using `operator bool` (proper is `ec != error_code{}`) which is not used in this project)